testmon/testmon_core.py

```diff
@@ -44,7 +45,7 @@ def blob_to_checksums(blob):
         a.frombytes(blob)
     except AttributeError:
         a.fromstring(blob)
-        return a.tolist()
+    return a
```

Comment:
This one change cut the memory usage in half:
![image](https://user-images.githubusercontent.com/596029/79508665-cd357c80-7fee-11ea-81da-922bd17ce8b7.png)